docs/src/main/sphinx/connector/faker.md

```diff
@@ -317,3 +338,41 @@ CREATE TABLE generator.default.customer (
   group_id INTEGER WITH (allowed_values = ARRAY['10', '32', '81'])
 );
 ```
+
+Faker can also automatically set the `default_limit` table property,
+and the `min`, `max`, `null_probability` column properties, based on statistics
+collected by scanning existing data, like in the following example:
+
+```sql
+CREATE TABLE generator.default.customer AS
+SELECT *
+FROM production.public.customer
+WHERE created_at > CURRENT_DATE - INTERVAL '1' YEAR;
+```
+
+Instead of using range, or other predicates, tables can be sampled,
+see {ref}`TABLESAMPLE<tablesample>`.
```

Comment:
whoa.. I didnt even know about that

change link to markdown syntax

Reply:
What's the Markdown syntax for referencing sections, not whole pages?

Reply:
Yup, not very well known. Using it for getting accurate stats without scanning whole data is a good example, maybe we can add it somewhere:
```sql
show stats for (select * from tpch.sf1.orders tablesample system(1));
```
I actually noticed this produces invalid stats in `tpch` - they're scaled down, since they're computed without scanning/generating any data.

Reply:
Syntax

```
[TABLESAMPLE](tablesample)
```